llmware/library.py

```diff
@@ -563,7 +814,24 @@ def export_library_to_jsonl_file(self, output_fp, output_fn, include_text=True,
 
     def pull_files_from_cloud_bucket (self, aws_access_key=None, aws_secret_key=None, bucket_name=None):
 
-        """ Pull files from private S3 bucket into local cache for further processing """
+        """ Pull files from private S3 bucket into local cache for further processing 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Pull files from private S3 bucket into local cache for further processing.
```